src/sql/src/plan/statement/ddl.rs

```diff
@@ -2337,12 +2461,95 @@ fn key_constraint_err(desc: &RelationDesc, user_keys: &[ColumnName]) -> PlanErro
     )
 }
 
-generate_extracted_config!(
-    CsrConfigOption,
-    (AvroKeyFullname, String),
-    (AvroValueFullname, String),
-    (NullDefaults, bool, Default(false))
-);
+/// Creating this by hand instead of using generate_extracted_config! macro
```

Comment:
Can you open an issue about this so we don't forget? I think keeping everything in the macro is good, so we should eventually add enum support to it.

Reply:
Sure 👍. I did not find any other similar use cases, so this could be an outlier.

Reply:
filed https://github.com/MaterializeInc/materialize/issues/22213, will add it in the comment as well.